frontend/src/libs/components/modal/styles.module.scss

```diff
@@ -0,0 +1,32 @@
+.modal {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  background: none;
+  border: none;
+}
+
+.modal::-webkit-scrollbar {
+  width: 0;
+}
+
+.modal::backdrop {
+  background: color(srgb 0 0 0 / 60%);
+}
+
+.modal-inner {
+  position: relative;
+  z-index: 10;
+  width: 940px;
+  height: 665px;
+  background-color: var(--white);
+  border: none;
+}
+
+.modal-pop-up-close {
+  position: absolute;
```

Comment:
Maybe let's better add header block to modal and just place close btn at the end of block?